ui/src/app/store/project/applications/applications.action.ts

```diff
@@ -0,0 +1,116 @@
+
```

Comment:
why this file is lost in app/store/project/application

it's named applications.action.ts, i think it can be in app/store directory